### PR TITLE
[tplinksmarthome] Bugfix missing colorTemperature channel update

### PR DESCRIPTION
**Pull request by Hilbrand**

### CODE REVIEW

File: addons/binding/org.openhab.binding.tplinksmarthome/README.md [hunk 1]
```diff
@@ -181,7 +181,7 @@ tplinksmarthome:lb130:home "Living Room Bulb 2" [ ipAddress="192.168.0.15", refr
 Switch   TP_L_Switch  "Switch"                            { channel="tplinksmarthome:hs100:home:switch" }
 Number   TP_L_RSSI    "Signal [%d] dB"           <signal> { channel="tplinksmarthome:hs100:home:rssi" }
 Dimmer   TP_LB_Bulb   "Dimmer [%d %%]"           <slider> { channel="tplinksmarthome:lb110:home:brightness" }
-Dimmer   TP_LB_ColorT "Color Temperature [%d] K" <slider> { channel="tplinksmarthome:lb130:home:color" }
+Dimmer   TP_LB_ColorT "Color Temperature [%d] %%" <slider> { channel="tplinksmarthome:lb130:home:colorTemperature" }
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.tplinksmarthome/README.md]: Was it already mentioned in the XML?

File: addons/binding/org.openhab.binding.tplinksmarthome/README.md [hunk 2]
```diff
@@ -181,7 +181,7 @@ tplinksmarthome:lb130:home "Living Room Bulb 2" [ ipAddress="192.168.0.15", refr
 Switch   TP_L_Switch  "Switch"                            { channel="tplinksmarthome:hs100:home:switch" }
 Number   TP_L_RSSI    "Signal [%d] dB"           <signal> { channel="tplinksmarthome:hs100:home:rssi" }
 Dimmer   TP_LB_Bulb   "Dimmer [%d %%]"           <slider> { channel="tplinksmarthome:lb110:home:brightness" }
-Dimmer   TP_LB_ColorT "Color Temperature [%d] K" <slider> { channel="tplinksmarthome:lb130:home:color" }
```
Comment (martinvw) [inline addons/binding/org.openhab.binding.tplinksmarthome/README.md]: Is this channel no longer needed / relevant?

File: addons/binding/org.openhab.binding.tplinksmarthome/README.md (hunk 1 above)
Comment (Hilbrand) [reply addons/binding/org.openhab.binding.tplinksmarthome/README.md]: During initial commit at some point the binding had a channel `colorTemperatureKelvin`. On this channel you could set color Temperature in Kelvin. There is also a color Temperature channel that works with percentages. Upon code review by @kaikreuzer this Kelvin channel was removed. I just noticed the readme contained a typo that can be traced back to this earlier implementation, and was incorrectly changed when removing that kelvin channel.

**Comment by martinvw** [comment]
Thanks!

**Comment by cweitkamp** [comment]
@Hilbrand Thank you. Does this PR fix #3883?

**Comment by Hilbrand** [comment]
@cweitkamp Yes it was intended to fix that problem. I hadn't seen the bug report yet :smile:

**Pull request merged**